src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/view.html.twig

```diff
@@ -32,55 +32,61 @@
 
 
 {% block content %}
-  <div class="row">
-
-    {% set displayAdminOrderTopHookContent = renderhook('displayAdminOrderTop', {'id_order': orderForViewing.id}) %}
+  <div id="order-view-page">
+    <div class="row d-print-none">
+{% set displayAdminOrderTopHookContent = renderhook('displayAdminOrderTop', {'id_order': orderForViewing.id}) %}
 
     {% if displayAdminOrderTopHookContent != '' %}
       <div class="col-md-12">
         {{ displayAdminOrderTopHookContent|raw }}
       </div>
     {% endif %}
-
-    <div class="col-md-12">
-      {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/order_actions.html.twig' %}
+      <div class="col-md-12">
+        {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/order_actions.html.twig' %}
+      </div>
     </div>
-  </div>
 
-  <div class="row d-none">
+    <div class="row d-none d-print-block mb-4">
     <div class="col-12" id="orderProductsModificationPosition"></div>
-  </div>
+      <div class="col-md-12">
+        {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/print_order_statistics.html.twig' %}
+      </div>
+    </div>
+
+    <div class="row d-none d-print-block mb-4">
+      <div class="col-md-12">
+        {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/print_title.html.twig' %}
+      </div>
+    </div>
 
-  <div class="row">
-    <div class="col-md-4">
-      {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/customer.html.twig' %}
+    <div class="row">
+      <div class="col-md-4 left-column">
+        {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/customer.html.twig' %}
 
       {{ renderhook('displayBackOfficeOrderActions', {'id_order': orderForViewing.id}) }}
-
-      {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/messages.html.twig' %}
+        {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/messages.html.twig' %}
 
       {{ renderhook('displayAdminOrderSide', {'id_order': orderForViewing.id}) }}
```

Comment:
```suggestion
        {{ renderhook('displayAdminOrderSide', {'id_order': orderForViewing.id}) }}
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17617/commits/880b0ffdaca18f032d2cb941f37e0751b4745479